node-template/runtime/Cargo.toml

```diff
@@ -48,3 +51,4 @@ std = [
 	"consensus-aura/std",
 	"offchain-primitives/std",
 ]
+no_std = []
```

Comment:
What is this `no_std` feature for?

Reply:
Every crate that should be build by the `wasm-builder` requires the `no_std` feature. This feature will be enabled by `wasm-builder` when building the WASM binary. 
For the normal runtime we for example need this to enable the `core` feature of `srml-contracts`.